<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CashAccountDialog</class>
 <widget class="QDialog" name="CashAccountDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>272</width>
    <height>196</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetDefaultConstraint</enum>
   </property>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="currentPathLabel">
       <property name="text">
        <string>Current path</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="currentPathLineEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="currencyLabel">
       <property name="text">
        <string>Currency</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="currencyComboBox"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="initialBalanceLabel">
       <property name="text">
        <string>Initial balance</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="initialBalanceDoubleSpinBox">
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="showGroupSeparator" stdset="0">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="positionLabel">
       <property name="text">
        <string>Position</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSpinBox" name="positionSpinBox">
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>currentPathLineEdit</tabstop>
  <tabstop>currencyComboBox</tabstop>
  <tabstop>initialBalanceDoubleSpinBox</tabstop>
  <tabstop>positionSpinBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
